src/styles/dotcms-theme/components/_calendar.scss

```diff
@@ -1,261 +1,310 @@
-  border-color: $error;
+    border-color: $error;
 }
 
 .p-calendar-w-btn {
-  border: 1px solid rgba(0, 0, 0, 0.38);
-  background: $white;
-  border-radius: 4px;
+    border: 1px solid rgba(0, 0, 0, 0.38);
+    background: $white;
+    border-radius: 4px;
 }
+
 .p-calendar-w-btn .p-inputtext {
-  background-image: none;
-  background: transparent;
-  border: 0 none;
+    background-image: none;
+    background: transparent;
+    border: 0 none;
 }
+
 .p-calendar-w-btn .p-inputtext:enabled:focus {
-  box-shadow: none;
+    box-shadow: none;
 }
+
 .p-calendar-w-btn .p-datepicker-trigger.p-button {
-  background-color: transparent;
-  border: 0 none;
+    background-color: transparent;
+    border: 0 none;
 }
+
 .p-calendar-w-btn .p-datepicker-trigger.p-button span {
-  color: rgba(0, 0, 0, 0.6);
+    color: rgba(0, 0, 0, 0.6);
 }
+
 .p-calendar-w-btn .p-datepicker-trigger.p-button:enabled:hover {
-  background: $bg-color-secondary;
+    background: $hover-bg;
 }
-.p-calendar-w-btn .p-datepicker-trigger.p-button:enabled:active, .p-calendar-w-btn .p-datepicker-trigger.p-button:focus {
-  background: $bg-color-secondary;
+
+.p-calendar-w-btn .p-datepicker-trigger.p-button:focus {
+    background: $focus-text-color;
 }
+
 .p-calendar-w-btn:not(.p-disabled):hover {
-  border-color: $black;
+    border-color: $black;
 }
+
 .p-calendar-w-btn:not(.p-disabled).p-inputwrapper-focus {
-  border-color:  $brand-primary;
-  box-shadow: inset 0 0 0 1px  $brand-primary, inset 0 0 0 1px  $brand-primary, inset 0 0 0 1px  $brand-primary, inset 0 0 0 1px  $brand-primary;
+    border-color: $brand-primary;
+    box-shadow: inset 0 0 0 1px $brand-primary, inset 0 0 0 1px $brand-primary, inset 0 0 0 1px $brand-primary, inset 0 0 0 1px $brand-primary;
 }
 
-
 .p-datepicker {
-  padding: $spacing-1;
-  background: $white;
-  color: $black;
-  border: 1px solid #e5e5e5;
-  border-radius: 4px;
+    padding: $spacing-1;
+    background: $white;
+    color: $black;
+    border: 1px solid #e5e5e5;
+    border-radius: 4px;
 }
+
 .p-datepicker:not(.p-datepicker-inline) {
-  background: $white;
-  border: 0 none;
-  box-shadow: 0 5px 5px -3px rgba(0, 0, 0, 0.2), 0 8px 10px 1px rgba(0, 0, 0, 0.14), 0 3px 14px 2px $bg-color-secondary;
+    background: $white;
+    border: 0 none;
+    box-shadow: 0 5px 5px -3px rgba(0, 0, 0, 0.2), 0 8px 10px 1px rgba(0, 0, 0, 0.14), 0 3px 14px 2px $bg-color-secondary;
 }
+
 .p-datepicker:not(.p-datepicker-inline) .p-datepicker-header {
-  background: $white;
+    background: $white;
 }
+
 .p-datepicker .p-datepicker-header {
-  padding: $spacing-1;
-  color: $black;
-  background: $white;
-  font-weight: 500;
-  margin: 0;
-  border-bottom: 1px solid $bg-color-secondary;
-  border-top-right-radius: 4px;
-  border-top-left-radius: 4px;
+    padding: $spacing-1;
+    color: $black;
+    background: $white;
+    font-weight: 500;
+    margin: 0;
+    border-bottom: 1px solid $bg-color-secondary;
```

Comment:
bg-color in a border, weird.

Reply:
This is for easier search & replace. Not final.